composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
I was thinking something along the lines of:

```php
public function __construct(
    ...,
    LoggerInterface = null
) { ... }
```

Along with

```php
if ($this->logger) {
    $this->logger->error(...);
}
```

In the handler itself.